setting_up_firely_server/deployment/deployment.rst

```diff
@@ -27,6 +19,55 @@ If you are not sure what would be the best option for your use case, or if you h
    Binaries <binaries>
    Docker <docker>
    Kubernetes / Helm <helm>
-   Azure <azureWebApp>
+   Azure Web App Service <azureWebApp>
    Reverse Proxy <reverseproxy/reverseProxy>
 
+.. rubric:: Local deployment
+
+**Advantages:**
+
+- Quick setup for development or evaluation purposes.
+- Ideal for developers testing Firely Server on their local machine.
+- No dependency on external infrastructure or orchestration tools.
+
+.. seealso:: 
+   See :ref:`use_binaries`
+
+------
+
+.. rubric:: Docker
+
+**Advantages:**
+
+- Simple and reproducible deployment using Docker images.
+- Minimal setup required
+- Easily integrates with CI/CD pipelines and container registries when integrating Firely Serve into existing products and workflows
+
+.. seealso:: 
+   See :ref:`use_docker`
+
+------
+
+.. rubric:: Azure Web App Service
+
+**Advantages:**
+
+- Fully managed hosting with built-in scaling, patching, and monitoring.
+- Easy deployment of containerized Firely Server with minimal DevOps overhead.
```

Comment:
Though possible to use a container, it seems that the main use case for App Service is to deploy an app from code. In the referenced section, we mention deploying a zip file, not a container...

I would instead create another deployment option: Azure Container App.
See e.g. this discussion for motivation/differences: https://www.davidyardy.com/blog/azure-appservices-containersapps

Reply:
Nice blog post, changed it to zip file